.gitignore

```diff
@@ -5,29 +5,28 @@
 node_modules
 misc
 coverage
-/.netlify
```

Comment:
This is #506. Unsure if it's fixed nowadays. If so, I guess we could keep this line and remove `.netlify` along with its comment.